testing/prow/benchmarking.sh

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env bash
+
+set -x
+set -o pipefail
+set -o errexit
+set -o nounset
+
+EPOCHS=3
+THRESHOLD=0.05
+MINSR=150.0
+MAXDR=30
+
+assess_benchmark_stats () {
+    artifact_dir=${ARTIFACT_DIR:-"/tmp/ci-artifacts_$(date +%Y%m%d)"}
+    sample_rate=$(cat "${artifact_dir}/benchmarking_run_ssd_sample_rate.log")
+    bench_duration=$(cat "${artifact_dir}/benchmarking_run_ssd_bench_duration.log")
+    if [[ $MINSR -gt $sample_rate ]] ; then
+        echo "Sample rate below minimum expected"
+        return 0
+    else
+        echo "Sample rate test passed!"
+    fi
+    if [[ $MAXDR -lt $bench_duration ]] ; then
+        echo "Benchmark duration above maximum expected"
+        return 0
+    else
+        echo "Benchmark duration test passed!"
+    fi
+    return 1
+}
```

Comment:
by convention, `0` means "everything's OK", and `not 0` means that an error occur

so here it would make more sense to have `return 1` in case of error.

Can you quote `$bench_duration` and `$sample_rate`? (mandatory as read from another file)
and `$MAXDR` + `$MINSR` for consistancy (not "mandatory", as we can visually asses its value)